src/App.jsx

```diff
@@ -1,31 +1,30 @@
 import { React, useEffect, useState } from 'react';
 import axios from 'axios';
+import Bars from 'react-loading-icons/dist/esm/components/bars';
 import WeatherProvider from './components/WeatherProvider';
 import { useTheme } from './components/ThemeProvider';
-import DailyWeather from './components/DailyWeather';
-import Sidebar from './components/Sidebar';
+import Sidebar from './components/sidebar/Sidebar';
+import MainDashboard from './components/main-dashboard/MainDashboard';
 
 const geocodingURI = 'http://api.openweathermap.org/geo/1.0/direct?q=';
 const weatherApiKey = process.env.REACT_APP_WEATHER_API_KEY;
 
 function App() {
-  const [geoData, setGeoData] = useState({});
+  const [geoData, setGeoData] = useState({
+    longitude: -122.674194,
+    latitude: 45.5202471,
+  });
```

Comment:
Instead of a useEffect to initialize the long/lat to Portland when the application is first loaded, I just added it right into the declaration of the useState hook. Much easier.

Reply:
You're right, that's much easier!